### PR TITLE
add configuration that allows documents to be uploaded to a patient's…

### PR DESCRIPTION
**Pull request by mogoodrich**
… record without being in the context of a visit/encounter

### CODE REVIEW

File: omod/src/main/java/org/openmrs/module/visitdocumentsui/page/controller/VisitDocumentsPageController.java [hunk 1]
```diff
@@ -46,6 +46,7 @@ public void controller(
 		jsonConfig.put("visit", visitWrapper == null ? null : convertVisit(visitWrapper.getVisit()));
 
 		jsonConfig.put("contentFamilyMap", getContentFamilyMap());
+		jsonConfig.put("associateWithVisitAndEncounter", context.associateWithVisitAndEncounter());
```
Comment (mogoodrich) [inline omod/src/main/java/org/openmrs/module/visitdocumentsui/page/controller/VisitDocumentsPageController.java:49]: I'm not crazy with this long-winded name "associatedWithVisitAndEncounter"... certainly open to better ideas!

**Comment by mogoodrich** [comment]
@mks-d  have you add a chance to take a look at this yet?  Thanks!

**Comment by mks-d** [comment]
@mogoodrich , sorry I could almost not do any tech stuff this week. Thanks for reminding me, I'll do my best to look at this before Monday.

**Comment by mogoodrich** [comment]
No worries, thanks @mks-d, just wanted to make sure I kept it on your radar!

**Review by mks-d — APPROVED** [review]

**Comment by mks-d** [comment]
Thanks a lot @mogoodrich!

**Comment by mogoodrich** [comment]
Thanks @mks-d!

**Pull request merged**